contracts/BridgeLimiter.sol

```diff
@@ -66,13 +54,18 @@ contract BridgeLimiter is
         return windowAmount + USDAmount > totalLimit;
     }
 
+    function willUSDAmountExceedLimit(uint256 amount) public view returns (bool) {
+        uint256 windowAmount = calculateWindowAmount();
+        return windowAmount + amount > totalLimit;
+    }
+
```

Comment:
nit: we can change to `willAmountExceedLimit`
```
    function willAmountExceedLimit(uint8 tokenId, uint256 amount) public view returns (bool) {
        uint256 windowAmount = calculateWindowAmount();	        uint256 windowAmount = calculateWindowAmount();
        uint256 USDAmount = calculateAmountInUSD(tokenId, amount);	        uint256 USDAmount = calculateAmountInUSD(tokenId, amount);
        return windowAmount + USDAmount > totalLimit;	        return windowAmount + USDAmount > totalLimit;
    }	    }
```

Reply:
Remember there are two function. `willAmountExceedLimit(uint8 tokenId, uint256 amount)` and `function willUSDAmountExceedLimit(uint256 amount)` both used for different purposes.